bootstrap/standalone/src/main/java/org/geysermc/platform/standalone/GeyserStandaloneBootstrap.java

```diff
@@ -62,22 +62,55 @@
 
     @Getter
     private boolean useGui = System.console() == null && !isHeadless();
+    private String configFilename = "config.yml";
 
     private GeyserConnector connector;
 
+
     public static void main(String[] args) {
-        for (String arg : args) {
+        GeyserStandaloneBootstrap bootstrap = new GeyserStandaloneBootstrap();
+        // Set defaults
+        boolean useGuiOpts = bootstrap.useGui;
+        String configFilenameOpt = bootstrap.configFilename;
+
+        for (int i = 0; i < args.length; i++) {
             // By default, standalone Geyser will check if it should open the GUI based on if the GUI is null
             // Optionally, you can force the use of a GUI or no GUI by specifying args
-            if (arg.equals("gui")) {
-                new GeyserStandaloneBootstrap().onEnable(true);
-                return;
-            } else if (arg.equals("nogui")) {
-                new GeyserStandaloneBootstrap().onEnable(false);
-                return;
+            String arg = args[i];
+            switch (arg) {
+                case "gui":
+                    useGuiOpts = true;
+                    break;
+                case "nogui":
+                    useGuiOpts = false;
+                    break;
```

Comment:
Maybe these should be moved to `--gui` and `--nogui` respectively, this will break backwards compatibility but will make them line up with the others better

Reply:
Or have both and a deprecation warning maybe?